packages/react-components/src/shared/src/useKeyboardNavigation.ts

```diff
@@ -14,7 +14,11 @@ interface KeyboardNavigationBindings {
     last: string[];
 }
 
-export function useKeyboardNavigation(focusManager: FocusManager, { previous = [], next = [], first = [], last = [] }: KeyboardNavigationBindings, { onSelect }: KeyboardNavigationOptions = {}) {
+interface KeyboardNavigationAPI {
```

Comment:
Usually for inputs (props) we usually suffix the interface name with "Props". For example, "TextInputProps".

Should we have a similar naming convention for return types?

Reply:
I feel this might be hard finding an appropriate standard for this. Maybe one will emerge naturally